src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -53,7 +53,11 @@ public class EditCommand extends Command {
 
```

Comment:
Remember to tweak the MESSAGE_USAGE as well since now edit can edit birthdays & other fields.

Reply:
agreed! thanks for pointing that out! fixed in e7540ca.